### PR TITLE
stripped back to story 1. Will display all entries in DB. Tested the …

### PR DESCRIPTION
**Pull request by Jago971**
…creation of div creation for each entry.≈

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -17,18 +17,6 @@
 // create DB with 3 items, 3 stats each ✅
 // fetch db and display ✅
```
Comment (mallowmew) [inline index.php]: Remove comments from production code

File: index.php [hunk 2]
```diff
@@ -17,18 +17,6 @@
...
-// $_GET form data ✅
-// move data to DB ✅
-
-// trainer tasks:
-
-// validation of inputs:
-// DB table for sizes, color, pattern ✅
-// dropdowns that populate colors, sizes, patterns ✅
-
 //----------------------------------------------------------------------------------------------------------------------required files
```
Comment (mallowmew) [inline index.php]: separators are unconventional and unnecessary, remove them

File: src/functions.php [hunk 3]
```diff
@@ -9,34 +9,6 @@ function connectDB() {
...
-    return $options;
-}
-//----------------------------------------------------------------------------------------------------------return full html dropdown - select, options, /select
-function createDropdown(string $name) : string {
-    $start = "<select name=\"{$name}\" id=\"{name}\"><option disabled=\"disabled\" selected=\"selected\">$name</option>";
-    $options = dropDownOptions(getTable($name, connectDB()));
-    $end = '</select><br><br>';
-    return $start . $options . $end;
-}
 //---------------------------------------------------------------------------------------------------------creates each sock div and ties together as one string
```
Comment (mallowmew) [inline src/functions.php]: again unnecessary separators / comments

The name of the function is `createSockDiv` do you need more explanation?

File: tests/createSockDivTest.php [hunk 4]
```diff
@@ -0,0 +1,54 @@
+<?php
+require_once 'src/functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class createSockDivTest extends TestCase {
+    public function testCreateSockDiv(): void
```
Comment (mallowmew) [inline tests/createSockDivTest.php]: what kind of test is this? success? fail? malformed?

File: tests/createSockDivTest.php [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        <p>Size: size2</p>
+        <p>Pattern: pattern2</p>
+        <p>Color: color2</p>
+        </div>";
+
+        $actual = createSockDiv($input);
+
+        $this->assertEquals($expected, $actual);
+    }
+    public function testCreateSockDivMalformedInputs(): void
```
Comment (mallowmew) [inline tests/createSockDivTest.php]: separate each test function with a blank line please

**Review by mallowmew — CHANGES_REQUESTED** [review]
Database:
You have other tables for the multiple choice / category-like fields, but you aren't using them. Please make at least the 'size' category require a JOIN to get the data, as you probably intended to do?

File: tests/createSockDivTest.php (hunk 4 above)
Comment (Jago971) [reply tests/createSockDivTest.php]: should I add the type into the test name eg. creatSockDivSuccessTest?

**Comment by Jago971** [comment]
@mallowmew Can you please review changes before merging? I have tidied up as requested and updated the displayCollection function to use a SQL query that relates all tables not just the sizes table

File: tests/createSockDivTest.php (hunk 4 above)
Comment (mallowmew) [reply tests/createSockDivTest.php]: yes, `createSockDivTest_Success` would be standard

**Pull request merged**